assets/components/contribute/contribute.jsx

```diff
@@ -13,12 +13,12 @@ import type { Node } from 'react';
 
 // ----- Types ----- //
 
-type PropTypes = {
+type PropTypes = {|
   copy: string | Node,
   children: Node,
   heading?: string,
```

Comment:
Is the exact object type declaration subordinate to the optional property type?

Reply:
Yes, in as much as if something is defined as optional then you don't have to pass it through, even if it is an exact type. The basic type ensures that everything that should be there, is there. The exact bolt on ensures that nothing extra is passed through. 

I think this line from the docs sums it up:

`it is not valid to pass an object with “extra” properties to an exact object type.`